src/demo-app/slide-toggle/slide-toggle-demo.scss

```diff
@@ -0,0 +1,8 @@
+.slide-toggle-example {
+  display:flex;
```

Comment:
Just to show that the slide toggle now has a display inline-block. If I do not use it, the browser will stretch the line and you can still click on the whole line to check the slide toggle.

[Example](http://plnkr.co/edit/iuTIRXgSXeQFWPopAgAA?p=preview)